miraclelinux/meta-emlinux
# Add PV to linux and linux-libc-headers recipes

# Purpose of pull request

Current linux-libc-headers version uses Linux kernels's git revision. However, it makes problem that linux-libc-header and linux-libc-header-k510 uses same revision eventhough they use different revision. So set PV to specify package version.

# Test
## How to test

Check its version by bitbake-layers command.

```
bitbake-layers show-recipes | grep "linux-libc-headers" -A1
```

Build core-image-minimal and check correct version was built.

## Test result

DISTRO=emlinux

```
masami@ubuntu1804:~/emlinux/qemuarm64-emlinux$ grep ^DISTRO conf/local.conf 
DISTRO ?= "poky"
DISTRO = "emlinux"
masami@ubuntu1804:~/emlinux/qemuarm64-emlinux$ bitbake-layers show-recipes | grep "linux-libc-headers" -A1
linux-libc-headers:
  meta                 5.0
linux-libc-headers-base:
  meta-debian          4.19
linux-libc-headers-k510:
  meta-emlinux         5.10
masami@ubuntu1804:~/emlinux/qemuarm64-emlinux$ ls tmp-glibc/work/aarch64-emlinux-linux | grep linux-libc-headers
linux-libc-headers-base/
masami@ubuntu1804:~/emlinux/qemuarm64-emlinux$ ls tmp-glibc/work/aarch64-emlinux-linux/linux-libc-headers-base/
./  ../  4.19-r0/
```

DISTRO=emlinux-k510

```
masami@ubuntu1804:~/emlinux/qemuarm64-k510$ grep ^DISTRO conf/local.conf
DISTRO ?= "poky"
DISTRO = "emlinux-k510"
masami@ubuntu1804:~/emlinux/qemuarm64-k510$ bitbake-layers show-recipes | grep "linux-libc-headers" -A1
linux-libc-headers:
  meta                 5.0
linux-libc-headers-base:
  meta-debian          4.19
linux-libc-headers-k510:
  meta-emlinux         5.10
masami@ubuntu1804:~/emlinux/qemuarm64-k510$ ls tmp-glibc/work/aarch64-emlinux-linux | grep linux-libc-headers
linux-libc-headers-k510/
masami@ubuntu1804:~/emlinux/qemuarm64-k510$ ls tmp-glibc/work/aarch64-emlinux-linux/linux-libc-headers-k510/
./  ../  5.10-r0/
```
